### PR TITLE
Add t_step parameter to frame reports

### PR DESCRIPTION
**Pull request by GianlucaFicarelli**
See NSETM-2194

### CODE REVIEW

**Review by joni-herttuainen — APPROVED** [review]
:metal:

File: bluepysnap/frame_report.py [hunk 1]
```diff
@@ -76,20 +76,28 @@ def _wrap_columns(columns):
...
 
-    def get(self, group=None, t_start=None, t_stop=None):
+    def get(self, group=None, t_start=None, t_stop=None, t_step=None):
         """Fetch data from the report.
 
         Args:
             group (None/int/list/np.array/dict): Get frames filtered by :ref:`Group Concept`.
             t_start (float): Include only frames occurring at or after this time.
             t_stop (float): Include only frames occurring at or before this time.
+            t_step (float): Optional time step, useful to reduce the number of samples.
```
Comment (mgeplf) [inline bluepysnap/frame_report.py:86]: I think it's important that this explains exactly what sort of stepping is done; ie, is it averaging skipped samples, etc.

Comment (GianlucaFicarelli) [reply bluepysnap/frame_report.py:86]: Do you think it's correct and clear enough in this way? Suggestions are welcome :)

Comment (mgeplf) [reply bluepysnap/frame_report.py:86]: yup, looks good

**Pull request merged**